Dropdown/Dropdown.module.less

```diff
@@ -0,0 +1,39 @@
+// Dropdown.module.less
+//
+@import "../styles/mixins.less";
+@import "../styles/skin.less";
+
+.Dropdown {
+	.applySkins({
+		.item {
+			border-color: gray;
+			border-style: solid;
+			border-width: 1px;
+		}
+		.transitionContainer {
+			position: absolute;
+			display: inline-block;
+			z-index: 1;
+			pointer-events: none;
+			min-width: 312px;
```

Comment:
We should probably have this as a less variable, along with all the other properties like the background color and border styles in `.dropdownList`, and the height in `.dropdownListWithScroller`